projects/subject-access-requests-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/SubjectAccessRequestsService.kt

```diff
@@ -14,8 +15,13 @@ class SubjectAccessRequestsService(private val personRepository: PersonRepositor
     }
 }
 
-fun PersonEntity.toPerson(): Person = Person(getName())
+fun PersonEntity.toPerson(): Person {
```

Comment:
i dont think you need the `let` here. Just use `listOfNotNull(secondName, thirdName).joinToString(" ")`

Reply:
this does not result in a null response, but a "" for the middle name.